libs/platform/ebpf_platform.h

```diff
@@ -26,6 +26,23 @@ extern "C"
         ((uint8_t*)(x)), sizeof((x)) - 1      \
     }
 
+#define EBPF_CACHE_LINE_SIZE (64)
+#define EBPF_CACHE_ALIGN_POINTER(P) (void*)(((uintptr_t)P + EBPF_CACHE_LINE_SIZE - 1) & ~(EBPF_CACHE_LINE_SIZE - 1))
+
+#define EBPF_DECLARE_ALIGNED_POINTER(TYPE, NAME)        \
```

Comment:
since they're declared as static below
```suggestion
#define EBPF_DECLARE_STATIC_ALIGNED_POINTER(TYPE, NAME)        \
```